### ISSUE TITLE
-Q doesn't quote metacharacters

### ISSUE THREAD

**Issue by benknoble** [beyondgrep/ack3#294]
I believe this is an ack 3.0 bug, since it did not exist before I upgraded.

Running `ack 'tags: ['` correctly notices the unmatched `[` in the regex.

Running `ack -Q 'tags: ['`, which should search for the fixed string `tags: [`, also produces this error. But `-Q` says:

```
Quote all metacharacters in PATTERN, it is treated as a literal.
```

A simpler reproduction case:

```
# ack -Q '*'
Regex: *
       ^---HERE Quantifier follows nothing in regex
```

Somewhat ironically, `ack -Q '\*'` (where I've quoted the metacharacters myself) searches for the exact string `\*`

**Comment by petdance** [comment]
Looks like this was OK in 3.2.0, but broke in 3.3.0.  Thanks for reporting.

**Comment by petdance** [comment]
This has been fixed at 57fb7da.  I'll be releasing v3.3.1 later today.

**Comment by benknoble** [comment]
Brilliant, thanks. Appreciate the quick turnaround :)
